drop-boxes/test-dropbox/script.py

```diff
@@ -77,7 +77,7 @@ def process(transaction):
         print qbicLocation
 
         # Update Sample Location
-        sampleTracker.updateSampleLocationToCurrentLocation(sampleID)
+        sampleTracker.updateSampleStatus(sampleID)
```

Comment:
here also? Update the variables and sample tracker to be able to call the new function?

Reply:
good catch, has been added now